rockcraft/lifecycle.py

```diff
@@ -159,13 +159,18 @@ def _pack(
     )
     emit.progress("Created new layer", permanent=True)
 
-    if project.entrypoint:
-        new_image.set_entrypoint(project.entrypoint)
-        if not project.cmd:
-            new_image.set_cmd([])
+    emit.progress("Adding Pebble entrypoint")
+    if project.services:
```

Comment:
isn't this field mandatory now?

Reply:
good point

Reply:
fixed

Reply:
oh actually no. `services` is optional, cause if you don't have it, the entrypoint will simply be `pebble enter`